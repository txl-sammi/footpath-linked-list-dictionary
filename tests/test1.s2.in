29.49
29.51
