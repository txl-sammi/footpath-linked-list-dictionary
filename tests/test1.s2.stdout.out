29.49 --> 29.5
29.51 --> 29.5
